### PR TITLE
docs: Cleanup obsolete EEA projects and update info about EEA main website

### PR DESCRIPTION
**Pull request by avoinea**

### CODE REVIEW

**Review by ichim-david — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,5 +165,8 @@ You should check the dependencies in their `package.json` for more details.
 - [plone.org.br](https://github.com/plonegovbr/plone.org.br) - Volto project for the [Brazilian Plone Community](https://plone.org.br)
 - [nsw-design-system-plone6-kit](https://github.com/pretagov/nsw-design-system-plone6-kit) - NSW Design System Plone 6 Kit Volto project for [NSW.gov.au sites](https://digitalnsw.pretagov.com.au/)
 - [volto-centraalmuseum-theme](https://github.com/intk/volto-centraalmuseum-theme) - Volto project for the [Centraal Museum & Rietveld](https://www.centraalmuseum.nl/nl) made for [INTK](https://www.intk.com/en).
+- [volto-eea-design-system](https://github.com/eea/volto-eea-design-system) - EEA Design System Plone 6 Kit Volto project for [European Environment Agency web sites](https://eea.github.io/volto-eea-design-system/)
+- [volto-eea-website-theme](https://github.com/eea/volto-eea-website-theme) - EEA Plone 6 Volto Theme for [European Environment Agency web sites](https://www.eea.europa.eu/en)
+- [volto-eea-kitkat](https://github.com/eea/volto-eea-kitkat) - A known good set of Volto addons to be used within all EEA projects and beyond made for [European Environment Agency](https://www.eea.europa.eu/en)
```
Comment (stevepiercy) [inline README.md:170]:
```suggestion
- [volto-eea-kitkat](https://github.com/eea/volto-eea-kitkat) - A known good set of Volto add-ons to be used within all EEA projects and beyond, made for [European Environment Agency](https://www.eea.europa.eu/en)
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor syntax, and it's good to go. Thank you!

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Let's port these changes over to Awesome-Volto, too: https://github.com/collective/awesome-volto/pull/20

**Comment by stevepiercy** [comment]
I ported the changes over to https://github.com/collective/awesome-volto/pull/20, and requested review of same.

**Pull request merged**